app/controllers/results_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ResultsController < ApplicationController
+  def index
+    @courses = Course
+      .includes(:provider)
+      .includes(:accrediting_provider)
+      .where(recruitment_cycle_year: Settings.current_cycle)
+      .where(provider_code: nil)
+      .page(params[:page])
+      .per(10)
```

Comment:
I don't know how this has been done already in this app, but shouldn't this be parameterised?

Reply:
Kaminari provider [a way](https://github.com/kaminari/kaminari#paginating-a-generic-array-object) to take a generic `Array` object and convert it into a paginated thing that can be used with Kaminari. 

I think that would give us a lot of pagination-related functionality for free, including stuff like a standardised view template. Would that be worth it?